unity-renderer/Assets/Scripts/MainScripts/DCL/Environment/Factories/ServiceLocatorFactory/ServiceLocatorFactory.cs

```diff
@@ -151,6 +154,12 @@ public static ServiceLocator CreateDefault()
                 new WearablesWebInterfaceBridge(),
                 featureFlagsDataStore));
 
+            result.Register<IEmotesService>(() => new EmotesService(new EmoteAnimationLoaderFactory(),
+                result.Get<IEmotesCatalogService>(),
```

Comment:
Despite it being widely used here resolving services while the container is not fully injected/being constructed is wrong design-wise. I propose to introduce variables instead.